### PR TITLE
Show the international format when displaying phone numbers in Valora

### LINKED ISSUE #5260: Leading 0 is automatically added to some international phone numbers + update Google phone lib package
### Expected Behavior
We should either keep the package from auto-completing with a 0 or let the user know that it's not something to worry about. It is probably also worthwhile to update the google-lib package to the latest release.

### Current Behavior
Appears that the package adds a 0 prefix to some international numbers when it shouldn't. Appears to largely be a display issue because the user still correctly receives the verification texts and is able to redeem from escrow.

https://celo-org.slack.com/archives/C018VLA3YAK/p1601038063011900
https://celo-org.slack.com/archives/CL7BVQPHB/p1601660582120600

### LINKED ISSUE #5261: Navigation drawer formats some countries' phone numbers incorrectly
### Expected Behavior
Drawer should consistently display the phone number using the domestic or international convention but not both.

### Current Behavior
The drawer displays the phone number as the international prefix + the domestic format. This is correct for some countries but not for others.

Example: https://celo-org.slack.com/archives/C018VLA3YAK/p1601385706015500?thread_ts=1601038090.012700&cid=C018VLA3YAK

### PR DESCRIPTION
**Pull request by tarikbellamine**
### Description
This PR changes it so that we show the international phone number when onboarding. Previously, we were attempting to do a combination of domestic and international when displaying, which resulted in the confusing outcomes detailed in the issues below.

### Other changes
Removed the international prefix from the drop down selector with the flag emojis so it was clear the prefix was not being double entered. Updated the GoogleLib package, which addresses some of the country specific issues we were seeing.

### Tested
Yes

### Related issues
- Fixes #5260
- Fixes #5261

### Backwards compatibility
Yes

IMG_0909

IMG_0910

IMG_0911

### CODE REVIEW

File: packages/mobile/src/onboarding/registration/NameAndNumber.tsx [hunk 1]
```diff
@@ -77,7 +77,8 @@ function getPhoneNumberState(
 
   if (phoneDetails) {
     return {
-      nationalPhoneNumber: phoneDetails.displayNumber,
+      // Show international display number to avoid confusion
+      nationalPhoneNumber: phoneDetails.displayNumberInternational,
```
Comment (jeanregisser) [inline packages/mobile/src/onboarding/registration/NameAndNumber.tsx]: I propose we also rename the field to further avoid confusion:
```suggestion
      internationalPhoneNumber: phoneDetails.displayNumberInternational,
```

File: packages/mobile/ios/Podfile.lock [hunk 2]
```diff
@@ -812,7 +812,7 @@ SPEC CHECKSUMS:
   FirebaseMessaging: 089b7a4991425783384acc8bcefcd78c0af913bd
   FirebaseStorage: 93fe2f8190a01bfb2b2c4932df7d679744c4ef1d
   Folly: b73c3869541e86821df3c387eb0af5f65addfab4
-  glog: 40a13f7840415b9a77023fbcae0f1e6f43192af3
+  glog: 1f3da668190260b06b429bb211bfbee5cd790c28
```
Comment (jeanregisser) [inline packages/mobile/ios/Podfile.lock]: Looks like these shouldn't be changed. CircleCI complains https://app.circleci.com/pipelines/github/celo-org/celo-monorepo/30707/workflows/95dd5695-2600-42f4-a85c-b0c7ceadbd1b/jobs/376203

File: packages/mobile/src/verify/VerificationEducationScreen.tsx [hunk 3]
```diff
@@ -235,7 +235,7 @@ function VerificationEducationScreen({ route, navigation }: Props) {
           label={t('nuxNamePin1:phoneNumber')}
           style={styles.phoneNumber}
           country={country}
-          nationalPhoneNumber={getE164DisplayNumber(e164PhoneNumber || '')}
+          nationalPhoneNumber={getDisplayNumberInternational(e164PhoneNumber || '')}
```
Comment (jeanregisser) [inline packages/mobile/src/verify/VerificationEducationScreen.tsx]:
```suggestion
          internationalPhoneNumber={getDisplayNumberInternational(e164PhoneNumber || '')}
```

File: yarn.lock [hunk 4]
```diff
@@ -18666,6 +18666,11 @@ google-gax@~1.12.0:
     semver "^6.0.0"
     walkdir "^0.4.0"
 
+google-libphonenumber@^3.2.15:
+  version "3.2.15"
+  resolved "https://registry.yarnpkg.com/google-libphonenumber/-/google-libphonenumber-3.2.15.tgz#3a01dc554dbf83c754f249c16df3605e5d154bb9"
+  integrity sha512-tbCIuzMoH34RdrbFRw5kijAZn/p6JMQvsgtr1glg2ugbwqrMPlOL8pHNK8cyGo9B6SXpcMm4hdyDqwomR+HPRg==
+
 google-libphonenumber@^3.2.4:
```
Comment (jeanregisser) [inline yarn.lock]: We should only have one instance of `google-libphonenumber`.

`yarn` has trouble keeping only one instance of compatible versions, when this happens you can manually delete both and re-run `yarn`.

**Review by jeanregisser — COMMENTED** [review]
Awesome! 💯 👍 
Left a couple of small comments.

**Comment by jeanregisser** [comment]
What happens when the user manually changes the country code from the text input field? 
Does it change the flag?

File: packages/mobile/ios/Podfile.lock (hunk 2 above)
Comment (tarikbellamine) [reply packages/mobile/ios/Podfile.lock]: Do you know why this happens? This occurs very frequently for me and I don't think I am the only one. I am just running the normal `bundle exec pod install` command and this is the result. Any ideas how we can resolve this for good?

**Comment by tarikbellamine** [comment]
> What happens when the user manually changes the country code from the text input field?
> Does it change the flag?

Good question. If the country code in the input field is manually changed, the "next" button will be disabled if the country code does not match the flag. So I don't think it is possible for there to be a mismatch.

File: packages/cli/package-lock.json [hunk 5]
```diff
@@ -17285,11 +17285,6 @@
         }
       }
     },
-    "google-libphonenumber": {
-      "version": "3.2.6",
-      "resolved": "https://registry.npmjs.org/google-libphonenumber/-/google-libphonenumber-3.2.6.tgz",
-      "integrity": "sha512-6QCQAaKJlSd/1dUqvdQf7zzfb3uiZHsG8yhCfOdCVRfMuPZ/VDIEB47y5SYwjPQJPs7ebfW5jj6PeobB9JJ4JA=="
-    },
```
Comment (jeanregisser) [inline packages/cli/package-lock.json:17292]: Surprised this gets deleted too here.
@gastonponti can you confirm?

**Review by jeanregisser — COMMENTED** [review]
Almost good, just concerned about the deletion of `google-libphonenumber` in the other lock files.

File: packages/mobile/ios/Podfile.lock (hunk 2 above)
Comment (jeanregisser) [reply packages/mobile/ios/Podfile.lock]: The Podspec content for `glog` is different between your machine and the CI machines.
Updating your local podspecs should fix it: `bundle exec pod install --repo-update`

File: packages/cli/package-lock.json (hunk 5 above)
Comment (gastonponti) [reply packages/cli/package-lock.json:17292]: mmm, yes it's weird. But technically we are using the `yarn.lock`, not the `package-lock.json`
And if you check the first lines its says:
```
  "name": "@celo/celocli",
  "version": "0.0.27",
```
I assume that was an `npm install` that someone commited some time ago
I strongly believe that this won't generate any problem. I would rule it out

File: packages/mobile/ios/Podfile.lock (hunk 2 above)
Comment (tarikbellamine) [reply packages/mobile/ios/Podfile.lock]: Unfortunately that doesn't fix it for me

**Review by gnardini — APPROVED** [review]
Everything looks good!

Just a little thing I noticed is that the `justifyContent: 'center',` on the `Expandable` file seems to have made many snapshots unrelated to this to change. Just mentioning to make sure that we take a look at those files to confirm that everything looks as expected :)

**Review by jeanregisser — APPROVED** [review]
+1 to what @gnardini  said above about the style.

**Comment by Lss-Ankit** [comment]
Hi @tarikbellamine we have verified this on latest Testflight build 1.8.0 (40) and android internal build 1.8.0 and observe the followings
Below are the 4 countries which we have verified internal format 

-  [x] User is able to see the international format on  account info page +63 for Philippines, +1 for US,+91 for india, +376 for Andorra

- [x] User is able to see the international format on Phone number page  +63 for Philippines, +1 for US,+91 for india, +376 for Andorra

- [x] User is able to see the international format on profile page by tapping on Hamburger menu  +63 for Philippines, +1 for US  0 for india, +376 for Andorra
Can you please let us know if we need to check for all countries for now we have verified with 4 countries.
Thanks!

**Comment by tarikbellamine** [comment]
Can you please try with Australia and Germany?

**Comment by Lss-Ankit** [comment]
Hi @tarikbellamine We have verified international format on Germany & Australia using latest Testflight 1.8.0 & Android internal build 1.8.0 and observe the followings.
 

- [x] User is able to see the international format on account info page +49 for Germany , +61 for Australia,
- [x]  User is able to see the international format on Phone number page  +49 for Germany , +61 for Australia,
- [x]  User is able to see the international format on profile page by tapping on Hamburger menu +49 for Germany , +61 for Australia,

Can you please let us know if we need to test for any other countries

**Comment by tarikbellamine** [comment]
That's all, thanks!

**Pull request merged**